test/test_eval.py

```diff
@@ -358,6 +358,44 @@ def test_extractive_qa_eval_multiple_queries(reader, retriever_with_docs, tmp_pa
     assert metrics["Retriever"]["ndcg"] == 0.5
 
 
+@pytest.mark.parametrize("retriever_with_docs", ["elasticsearch"], indirect=True)
+@pytest.mark.parametrize("document_store_with_docs", ["elasticsearch"], indirect=True)
```

Comment:
Wouldn't an InmemoryDocumentStore + tfidf (or even mock retriever) be sufficient here?

Reply:
Filters aren't implemented in tfidf retriever. I prefer using elasticsearch here to test the filtering functionality.